pkg/sql/sem/tree/cast.go

```diff
@@ -1020,6 +1020,14 @@ var castMap = map[oid.Oid]map[oid.Oid]cast{
 		oid.T_text:    {maxContext: CastContextAssignment, origin: contextOriginAutomaticIOConversion, volatility: VolatilityImmutable},
 		oid.T_varchar: {maxContext: CastContextAssignment, origin: contextOriginAutomaticIOConversion, volatility: VolatilityImmutable},
 	},
+	oid.T__uuid: {
```

Comment:
is this still meant to be here?
edit: nvm it's in the next commit, up to you if you want to fix it

Reply:
I botched the rebase. Fixed.